### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by revolter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ the JSON file's path.
 + `icon-size` (number, optional) - Size of all the icons inside the DMG
 + `window` (object, optional) - Window options
     + `position` (object, optional) - Position when opened
-        + `x` (number, required) - X position relative to bottom of the screen
+        + `x` (number, required) - X position relative to right of the screen
```
Comment (tylerlong) [inline README.md]: Are your sure it's right instead of left? I've never heard of any coordinate system that starts from right.

Comment (LinusU) [inline README.md]: Yeah, I also think it should be "left"

Comment (revolter) [inline README.md]: So the next line should be `top` too then.

Comment (LinusU) [inline README.md]: No, I believe that "bottom" is correct

Comment (revolter) [inline README.md]: Oh, so the origin is in the bottom left, right? I'll amend my commit in a couple of hours.

Comment (LinusU) [inline README.md]: ping @revolter 💌

Comment (revolter) [inline README.md]: Wow, really sorry about this.

**Comment by LinusU** [comment]
Thank you!

**Pull request merged**